src/main/kotlin/org/wfanet/panelmatch/client/deploy/example/gcloud/BUILD.bazel

```diff
@@ -23,18 +23,12 @@ kt_jvm_library(
 java_binary(
     name = "GoogleCloudExampleDaemonMain",
     main_class = "org.wfanet.panelmatch.client.deploy.example.gcloud.GoogleCloudExampleDaemonMainKt",
-    runtime_deps = [
-        ":gcloud",
-        "//imports/java/com/google/common:tink_gcpkms",
-    ],
+    runtime_deps = [":gcloud"],
```

Comment:
Do we confirm tink is running without runtime_deps? That was not the case when I was deploying.